cuda/ubi8-python-3.8/Dockerfile

```
@@ -81,6 +81,48 @@ RUN yum install -y \
     && yum clean all \
     && rm -rf /var/cache/yum/*
 
+# Install CUDA devel from:
+# https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/11.4.2/ubi8/devel/Dockerfile
+ENV NV_NVPROF_VERSION 11.4.120-1
+ENV NV_NVPROF_DEV_PACKAGE cuda-nvprof-11-4-${NV_NVPROF_VERSION}
+ENV NV_CUDA_CUDART_DEV_VERSION 11.4.108-1
+ENV NV_NVML_DEV_VERSION 11.4.120-1
+ENV NV_LIBCUBLAS_DEV_VERSION 11.6.1.51-1
+ENV NV_LIBNPP_DEV_VERSION 11.4.0.110-1
+ENV NV_LIBNPP_DEV_PACKAGE libnpp-devel-11-4-${NV_LIBNPP_DEV_VERSION}
+ENV NV_LIBNCCL_DEV_PACKAGE_NAME libnccl-devel
+ENV NV_LIBNCCL_DEV_PACKAGE_VERSION 2.11.4-1
+ENV NCCL_VERSION 2.11.4
+ENV NV_LIBNCCL_DEV_PACKAGE ${NV_LIBNCCL_DEV_PACKAGE_NAME}-${NV_LIBNCCL_DEV_PACKAGE_VERSION}+cuda11.4
+
+RUN yum install -y \
+    make \
+    cuda-command-line-tools-11-4-${NV_CUDA_LIB_VERSION} \
+    cuda-libraries-devel-11-4-${NV_CUDA_LIB_VERSION} \
+    cuda-minimal-build-11-4-${NV_CUDA_LIB_VERSION} \
+    cuda-cudart-devel-11-4-${NV_CUDA_CUDART_DEV_VERSION} \
+    ${NV_NVPROF_DEV_PACKAGE} \
+    cuda-nvml-devel-11-4-${NV_NVML_DEV_VERSION} \
+    libcublas-devel-11-4-${NV_LIBCUBLAS_DEV_VERSION} \
+    ${NV_LIBNPP_DEV_PACKAGE} \
+    ${NV_LIBNCCL_DEV_PACKAGE} \
+    && yum clean all \
+    && rm -rf /var/cache/yum/*
+
+ENV LIBRARY_PATH /usr/local/cuda/lib64/stubs
+
+# Install CUDA devel cudnn8
+# https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/11.4.2/ubi8/devel/cudnn8/Dockerfile
+ENV NV_CUDNN_VERSION 8.2.4.15-1
+ENV NV_CUDNN_PACKAGE libcudnn8-${NV_CUDNN_VERSION}.cuda11.4
+ENV NV_CUDNN_PACKAGE_DEV libcudnn8-devel-${NV_CUDNN_VERSION}.cuda11.4
+
+RUN yum install -y \
```

Missing label from https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/11.4.2/ubi8/devel/cudnn8/Dockerfile#L29

```suggestion
ENV NV_CUDNN_PACKAGE_DEV libcudnn8-devel-${NV_CUDNN_VERSION}.cuda11.4

LABEL com.nvidia.cudnn.version="${NV_CUDNN_VERSION}"

RUN yum install -y \
```